Signal/src/AppDelegate.m

```diff
@@ -363,6 +369,9 @@ - (void)showErrorAlertWithTitle:(NSString *)title message:(NSString *)message
     [controller addAction:[UIAlertAction actionWithTitle:NSLocalizedString(@"OK", nil)
                                                    style:UIAlertActionStyleDefault
                                                  handler:nil]];
+    [[Environment getCurrent].signalsViewController.presentedViewController presentViewController:controller
```

Comment:
I'm running in circles over here. The real issue is that presentedViewController is nil.

Here's how I fixed it: https://github.com/WhisperSystems/Signal-iOS/commit/1b61c3b0ce8dc6f083f4ad749be53b8c8509fea4

If that LGTY, feel free to merge away.